src/app/components/TicketCount.tsx

```diff
@@ -1,26 +1,30 @@
 import { FC } from 'react';
 
 type Props = {
-  ticketCount: number;
-  onClickMinus: () => void;
-  onClickPlus: () => void;
+	ticketCount: number;
```

Comment:
It looks like you have inconsistently configured formatting. Avoid committing changes that only change formatting because that will make review harder and it also makes it harder to look back at changes when trying to figure out the causes of future bugs.